### ISSUE TITLE
calculate the accuracy of model 

### ISSUE THREAD

**Issue by LeTrungLinh** [niuwagege/LDRNet#1]
Thank for your research. I implement your opensource , i want to calculate the accuracy of model and evaluate the accuracy of your DL model . Can you give me the method or code ?. Moreover i want to convert it to TensorRT. Thank you so much !

**Comment by niuwagege** [comment]
The test.py is updated.

**Comment by LeTrungLinh** [comment]
I'll check it and try to run . Thank you so much!

**Comment by LeTrungLinh** [comment]
Hello, can you give me the checkpoint by pytorch or onnx file, or file train in pytorch . I used to train your model on pytorch and replace backbone but maybe my dataset is small and just reach 92% accuracy. Thank for your help ! <3

**Comment by niuwagege** [comment]
you can use tf2onnx convert the tf2 model to onnx.

python -m tf2onnx.convert --saved-model tensorflow-model-path --output model.onnxonnx/tensorflow-onnxpython -m tf2onnx.convert --saved-model tensorflow-model-path --output model.onnx
